README.md

```diff
@@ -49,6 +49,91 @@ These widgets can be customized, added, removed and even reordered
 - Notifications (Will always be visible)
 - Label
 - Mpris (Media player controls for Spotify, Firefox, Chrome, etc...)
+Volume Controll (using pamixer)
+- Brightness Controll (using brightnessctl)
+- Controllcenter with Powermenu, Powermode-Menu and additional buttons
+
+Example Config: 
+```json
+{
+  ...
+  "brightness": {
+    "label": "",
+    "device": "intel_backlight"
+  },
+  "volume": {
+    "label": ""
+  },
+  "controlls": {
+    "power-buttons": {
+      "type": "menu",
+      "label": "right", 
+      "position": "topbar-right",
+      "actions": [ 
+        {
+          "label": " Reboot",
+          "command": "systemctl reboot"
+        },
+        {
+          "label": " Lock",
+          "command": "swaylock -f --image ~/.config/wallpapers/snow_woods.jpg"
+        },
+        {
+          "label": " Logout",
+          "command": "swaymsg exit"
+        },
+        {
+          "label": " Shut down",
+          "command": "systemctl poweroff"
+        }
+      ]
+    },
+    "powermode-buttons": {
+      "type": "menu",
+      "label": "left", 
+      "position": "topbar-left",
+      "actions": [ 
+        {
+          "label": "Performance",
+          "command": "powerprofilesctl set performance"
+        },
+        {
+          "label": "Balanced",
+          "command": "powerprofilesctl set balanced"
+        },
+        {
+          "label": "Power-saver",
+          "command": "powerprofilesctl set power-saver"
+        }
+      ]
+    },
+    "controll-buttons": {
+      "type": "buttons",
+      "actions": [
+        {
+          "label": "直",
+          "command": "~/.config/rofi/rofi-wifi-menu.sh"
+        },
+        {
+          "label": "",
+          "command": "~/.config/rofi/rofi-bluetooth"
+        }
+      ]
+    },
+    "topbar-buttons": {
+      "type": "buttons",
+      "position": "topbar-right",
+      "actions": [
+        {
+          "label": "",
+          "command": "...",
+        }
+      ]
+    }
+  }
+  ...
+}
```

Comment:
Hmm, ok. Since the menus (and also the buttons if you choose `"position": "topbar-right"`) will be placed into the same `topbar_container` I will have to edit the `controlCenter.ui`. I'll check it out :)

Reply:
Oh, there's positioning! Didn't notice... 😅

I wonder if there's a better way of doing this than my suggestion above?

Reply:
Maybe something like this?

```json
"widgets": [
  "menubar#label",
  "buttons",
  "buttons#AnotherButton123",
  "title",
  "dnd",
  "mpris",
  "notifications"
],
"widget-config": {
  "menubar#label": {
    "menus": [
      {
        "label": "Generic menu item",
        "position": "right",
        "actions": [ 
          {
            "label": " Reboot",
            "command": "systemctl reboot"
          },
          ...
        ]
      },
      {
        "label": "Another menu config",
        "position": "left",
        "actions": [ 
          {
            "label": " Reboot",
            "command": "systemctl reboot"
          },
          ...
        ]
      },
    ]
  },
  "buttons": {
    "actions": [
      {
        "label": "直",
        "command": "~/.config/rofi/rofi-wifi-menu.sh"
      },
      ...
    ]
  },
  "buttons#AnotherButton123": {
    "actions": [
      {
        "label": "直",
        "command": "~/.config/rofi/rofi-wifi-menu.sh"
      },
      ...
    ]
  },
}
```

Would that work?

Reply:
Jup, that shoud work. I would add an additional type to the menu so you could add buttons, which would not open a dropdown (e.g. the screenshot button in my config). Will change it :)

Reply:
Cool! Thanks for your patience 😅

Reply:
Done. I might have to add some more checks when parsing the config, but then it should be ready :)